formatTest/typeCheckedTests/expected_output/attributes.re

```diff
@@ -306,7 +309,6 @@ let myFun
 
 let myFun
     (X (hello [@onHello]) | Y (hello [@onHello])) => hello;
-
 /* Another bug: Cannot have an attribute on or pattern
-   let myFun = fun ((X hello | Y hello) [@onOrPattern]) => hello;
-   */
+let myFun = fun ((X hello | Y hello) [@onOrPattern]) => hello;
```

Comment:
Interesting.

The input to this is:
![image](https://cloud.githubusercontent.com/assets/1017689/17000288/a8147418-4e76-11e6-9705-eaf15632b078.png)

I don't have any special logic yet to reformat multiline comments. I originally thought it's good to maintain the original format so it doesn't mess up with people drawing ascii arts in comments.